### ISSUE TITLE
Language switcher in post/page editor disappeared with Gutenberg 2.8.0 update

### ISSUE THREAD

**Issue by Theremingenieur** [VaLeXaR/wp-multilang#53]
While with Gutenberg 2.7.0, the language switcher on the top left was there in the post/page editor, only annoying that its Z-Order was always behind the status messages ("Your post has been saved"), it is not longer visible after the 2.8.0 update.
![capture d ecran 2018-05-07 a 10 41 22](https://user-images.githubusercontent.com/25463159/39693265-c04df53c-51e3-11e8-9651-1cc002f7f53f.png)

**Comment by ikevinsolomon** [comment]
Yes, I've encountered this happens because of the sequence of the JS being loaded, I created a new action to enqueue a script with the position as last / or you can change the function to be invokes on on page load instead of the document.ready

**Comment by ikevinsolomon** [comment]
find this file classs-wpm-gutenberg.php modify it to : 
```
$(window).on('pageshow',function(){
	   		console.log('OnPageShow');
	   		wp.api.init().then(function() {
	   			if ($("#wpm-language-switcher").length === 0) {
	   				var language_switcher = wp.template("wpm-ls-customizer");
	   				console.log(language_switcher);
	   				$(".edit-post-header-toolbar").append(language_switcher);
	   			}
	   		});

	   	});
```
This should fix the issue :)

**Comment by VaLeXaR** [comment]
Thank @ikevinsolomon . This is good solution. I will add this in the next version.

**Comment by ikevinsolomon** [comment]
@VaLeXaR I did, is there something i'm missing ? can you please elaborate on how to enable the plugin for it ?
